### DISCUSSION TITLE
Is the .svg extension in the pathname an officially supported feature?

### DISCUSSION THREAD

**Discussion by igordanchenko** [badges/shields#9684] [Q&A] [answered]
I looked through the documentation but didn't find an answer to this question. Is the `.svg` extension in the pathname an officially supported feature?

For example, the following is the default badge displaying the NPM package license:

https://img.shields.io/npm/l/react-photo-album?color=blue

I tried adding the `.svg` extension to the pathname and it also works:

https://img.shields.io/npm/l/react-photo-album.svg?color=blue

Can somebody confirm this is an officially supported feature?

**Additional context**

The reason why I'm looking to add `.svg` extension to the pathname is due to [Google Chrome Lighthouse](https://github.com/GoogleChrome/lighthouse) failing image size audit with "Serves images with low resolution" audit item. This is happening because the Lighthouse tries to [guess](https://github.com/GoogleChrome/lighthouse/blob/b129c136bff66c6c74c17d92a61c8c245abca435/core/audits/image-size-responsive.js#L99-L101) image MIME type from its URL. So when it doesn't see an `.svg` extension in the URL, it assumes it's a raster image.

**Comment by chris48s** [comment, marked as answer]
Yes. Extension-less is the default, but `.svg` isn't going anywhere.

**Reply by igordanchenko** [reply]
Thank you!
